test/Swashbuckle.AspNetCore.Newtonsoft.Test/SchemaGenerator/NewtonsoftSchemaGeneratorTests.cs

```diff
@@ -287,6 +287,37 @@ public void GenerateSchema_SetsNullableFlag_IfPropertyIsReferenceOrNullableType(
         Assert.Equal(expectedNullable, schema.Properties[propertyName].Nullable);
     }
 
+    [Theory]
+    [InlineData(typeof(TypeWithNullableProperties), nameof(TypeWithNullableProperties.NullableIntEnumProperty), false)]
```

Comment:
Does this need to be a `[Theory]` if there's only one test case? Same for the test below.

Reply:
Not in particular, originally I had started changing the test above it but I wanted to add an extra assertion for the schema reference id.

Reply:
Yeah I figured it was probably an artifact from some refactoring - let's just make them both a `[Fact]`.